easybuild/easyconfigs/p/PGI/PGI-16.1-GCC-4.9.2-2.25.eb

```diff
@@ -0,0 +1,26 @@
+name = 'PGI'
+version = '16.1'
+
+homepage = 'http://www.pgroup.com/'
+description = "C, C++ and Fortran compilers from The Portland Group - PGI"
+
+toolchain = {'name': 'dummy', 'version': ''}
+
+sources = ['pgicdk-%(version_major)s%(version_minor)s.tar.gz']
```

Comment:
@boegel: Apologies for the delay. I'm told the PGI CDK ("Cluster Development Kit") includes some advanced packages that are not included with the basic PGI tarball. What way of handling this PGI variant would be most EasyBuild-ish?